drivers/rn2xx3/rn2xx3_netdev.c

```diff
@@ -0,0 +1,359 @@
+/*
+ * Copyright (C) 2017
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_rn2xx3
+ * @{
+ * @file
+ * @brief       Netdev adaptation for the rn2xx3 driver
+ *
+ * @author      Alexandre Abadie <alexandre.abadie@inria.fr>
+ * @}
+ */
+
+#include <stddef.h>
+#include <string.h>
+#include <errno.h>
+
+#include "net/netopt.h"
+#include "net/netdev.h"
+#include "net/loramac.h"
+
+#include "rn2xx3_internal.h"
+#include "rn2xx3.h"
+
+#define ENABLE_DEBUG (0)
+#include "debug.h"
+
+/* Internal helper functions */
+static int _set_state(rn2xx3_t *dev, netopt_state_t state);
+static int _get_state(rn2xx3_t *dev, void *val);
+
+/* Netdev driver api functions */
+static int _send(netdev_t *netdev, const struct iovec *vector, unsigned count);
+static int _recv(netdev_t *netdev, void *buf, size_t len, void *info);
+static int _init(netdev_t *netdev);
+static void _isr(netdev_t *netdev);
+static int _get(netdev_t *netdev, netopt_t opt, void *val, size_t max_len);
+static int _set(netdev_t *netdev, netopt_t opt, const void *val, size_t len);
+
+const netdev_driver_t sx127x_driver = {
```

Comment:
What is this? `sx127_*` code does not belong here. It looks like copy-paste issue.